src/iterate.js

```diff
@@ -151,6 +163,10 @@ class Iterate extends events.EventEmitter {
       // Wait until we've done all the checks to emit success
       this.emit('iteration-success', value);
       debug(`ran handler in ${diff} seconds`);
+      if (this.monitor) {
+        this.monitor.measure('iteration-time', diff * 1000);
+        this.monitor.count('successful-iteration', 1);
+      }
```

Comment:
it's not a comment that we shouldn't use ms, it's that when we generate the diff, we do Date - Date / 1000 to get seconds, but then multiply that value back by 1000 to get it back into ms.  I wonder if we'd be better just leaving the diff in MS and do the comparison in a way that it supports ms instead of s.